model/firewall.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require_relative "../model"
+
+class Firewall < Sequel::Model
+  one_to_many :firewall_rules, key: :firewall_id
+  many_to_one :vm, key: :vm_id
+
+  include ResourceMethods
```

Comment:
should this go at the top? it's a pretty important thing about it. Unless it's on model...which I don't think it is. If you have reasons why it's there, let me know, my opinion the subject is not particularly rigid, I'd be curious how you are thinking about that.

Reply:
I guess, you mean that `include ResourceMethods` should go below `plugin :association_dependencies`. This was mostly an error from me. Now, I realize everywhere in the code it's after the plugin. However, to be honest, I don't know what difference this does make. Can you explain? I have switched them btw.